tests/functionnal/Project.php

```diff
@@ -397,5 +397,63 @@ public function testClone()
 
        // Compare teams
         $this->array($team_clone)->isEqualTo($team);
+
+        // Add a task to project
+        $task1_name = 'Project testClone - Task' . mt_rand();
+        $this->createItems('ProjectTask', [
+            [
+                'projects_id' => $projects_id,
+                'name'        => $task1_name,
+            ],
+        ]);
+        $task1_id = getItemByTypeName("ProjectTask", $task1_name, true);
+
+        // Load current task
+        $task1 = new \ProjectTask();
+        $this->boolean($task1->getFromDB($task1_id))->isTrue();
```

Comment:
You can simplify your code here.

```suggestion
        $task1 = $this->createItem('ProjectTask', [
            'projects_id' => $projects_id,
            'name'        => $task1_name,
        ]);
        $task1_id = $task1->fields['id'];
```